Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -202,7 +233,9 @@ int VertexLoaderX64::ReadVertex(OpArg data, VertexComponentFormat attribute, Com
         dest.AddMemOffset(sizeof(float));
 
         // zfreeze
-        if (native_format == &m_native_vtx_decl.position)
+        if (native_format == &m_native_vtx_decl.position ||
+            native_format == &m_native_vtx_decl.normals[1] ||
```

Comment:
Doesn't have to be this review, but could (should?) this be `normal`, `binormal` and `tangent`?  Are we gaining anything by them being in an array?

Reply:
Seems like the `if (vert_decl.normals[1].enable)` would read more clearly below as `if (vert_decl.binormal.enable)`.

(I know I'm nitpicking a bit here)

Reply:
It's convenient to be able to iterate over them because they all share the same format, but it's a lot more messy now that special-casing is needed (on master there isn't any situation where indices 1 and 2 were directly referenced like this).